tests/test_other.py

```diff
@@ -8684,8 +8684,8 @@ def test_js_optimizer_parse_error(self):
 var ASM_CONSTS = [function() { var x = !<->5.; }];
                                         ^
 ''', '''
-var ASM_CONSTS = [function() {var x = !<->5.;}];
-                                       ^
+  0: function() {var x = !<->5.;}
+                          ^
```

Comment:
I think this `^` mark should move, as the `<` it points to is now more to the left?